tests/users/test_api.py

```diff
@@ -334,7 +334,7 @@ async def test_create_first_user(
     client = await spawn_client()
 
     if not first_user_exists:
-        await mongo.users.delete_many({})
+        await data_layer.users.delete_all()
```

Comment:
imo I would leave this as it was. I can't see a practical use for `delete_all` in production and triggering it by accident would be catastrophic

Reply:
needed to also delete postgres Users, so I have just moved delete all into this function as it was the only place it was called.